integration-tests/tests/billing-projects.js

```diff
@@ -28,11 +31,9 @@ const billingProjectsPage = (testPage, testUrl) => {
       await click(testPage, clickable({ text: 'Owners' }))
     },
 
-    selectProject: async billingProjectName => {
-      await noSpinnersAfter(
-        testPage,
-        { action: () => click(testPage, clickable({ text: billingProjectName })) }
-      )
+    selectProject: async (billingProjectName, cloudPlatform = GCP) => {
+      const text = cloudPlatform === GCP ? `Google Cloud Platform${billingProjectName}` : `Microsoft Azure${billingProjectName}`
```

Comment:
This is because the icon is now shown before the project name, within the link. This was introduced in https://github.com/DataBiosphere/terra-ui/commit/0a6eb0ce603844fef4e634595bc11f54d9185d55, but the test wasn't impacted at that time because it uses mock Ajax data, and the data wasn't updated to include the cloudPlatform.